core_chat_presentation/src/commonMain/kotlin/net/thechance/mena/core_chat/presentation/utils/Mapper.kt

```diff
@@ -0,0 +1,68 @@
+package net.thechance.mena.core_chat.presentation.utils
+
+import kotlinx.datetime.*
+import net.thechance.mena.core_chat.presentation.screen.messaging.ChatListItem
+import net.thechance.mena.core_chat.presentation.screen.messaging.MarkedMessageUiState
+import net.thechance.mena.core_chat.presentation.screen.messaging.MessageUiState
+
+
+fun List<MessageUiState>.markLastInSeries(): List<MarkedMessageUiState> {
```

Comment:
we need to follow one style for adding mappers in our feature
plz move theme to the ui state